core/core_integrations/slack.js

```diff
@@ -1,12 +1,10 @@
 var request = require.safe('request'),
 shim = require.once('../shim.js'),
 WebSocket = require.safe('ws'),
-async = require.safe('async'),
-sleep = require.safe('sleep'),
+deasync = require.safe('deasync'),
 platform = null,
 sockets = [],
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2571707054/issues/source?bid=3282987&fileBranchId=3288593#l6)